src/renderer/components/FrequencyBand.tsx

```diff
@@ -42,21 +43,45 @@ const FrequencyBand = forwardRef(
     { sliderIndex, filter, isMinSliderCount }: IFrequencyBandProps,
     ref: ForwardedRef<HTMLDivElement>
   ) => {
+    const INTERVAL = 100;
     const { globalError, setGlobalError, dispatchFilter } = useAquaContext();
     const [isLoading, setIsLoading] = useState(false);
     const isRemoveDisabled = useMemo(
       () => isMinSliderCount || isLoading,
       [isLoading, isMinSliderCount]
     );
 
+    const throttleSetGain = useThrottleFuture(async (newValue: number) => {
+      // Always dispatch first so that we don't see jitter in the sliders.
```

Comment:
minor nit: could replace this with a `/* */` block considering how long this comment chunk is. It's also fine to leave it as is tho